reference/storage/file-signUrl.md

```diff
@@ -0,0 +1,60 @@
+Create a signed url for access to file
+
+```javascript
+import { bucket, FileMode } from '@nitric/sdk';
+
+const assets = bucket('assets').for('reading', 'writing');
+
+const logo = assets.file('images/logo.png');
+
+const logoUrl = await logo.signUrl(FileMode.Read);
+```
+
+## Parameters
+---
+
+**mode** required `FileMode`
+
+The mode the signed url with operate in.
+
+Available options are:
+ - `FileMode.Read`
+ - `FileMode.Write`
+
+---
+
+**options** `SignUrlOptions`
+
+Properties
+
+| name   | type     | default | description               |
+| ------ | -------- | ------- | ------------------------- |
+| expiry | `number` | 600     | seconds until link expiry |
+
+---
+
+## Examples
+
+Create a readable link that is valid for the next 10 minutes
+```javascript
```

Comment:
Minor thing, there should probably be an extra line between the text and code section - some markdown renderers get funny about this.

Reply:
done